app/src/main/res/values/dimens.xml

```diff
@@ -47,5 +47,7 @@
     <dimen name="viewer_toolbar_title">16sp</dimen>
 
     <dimen name="quote_collapsed_height">32dp</dimen>
+    <dimen name="rc_16dp">16dp</dimen>
```

Comment:
What `rc` stands for?

Reply:
Rc stands for rocket chat. My idea is to give generic names to these dimensions. It could be resolved by naming it 16dp instead of rc_16dp.